.env.example

```
@@ -28,6 +28,15 @@ ZFS_HOST_KEY_PASSPHRASE_1=""
 # ZFS_HOST_USER_2=""
 # ZFS_HOST_PASSWORD_2=""
 
+# Proxmox VE API Configuration
+# Token format: USER@REALM!TOKENID (e.g., root@pam!monitoring)
+# Create tokens via Datacenter > Permissions > API Tokens in Proxmox UI
+PROXMOX_HOST=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)